subsys/net/ip/tcp_internal.h

```diff
@@ -173,6 +173,11 @@ struct net_tcp {
 	 */
 	u16_t recv_wnd;
 
+	/**
+	 * Zero TCP receive window value sent to the peer.
+	 */
+	bool zero_wnd_sent;
```

Comment:
Not sure about the feature itself, but there are 13 unused bits in _padding that could be used for this boolean.